packages/autorest.python/autorest/codegen/__init__.py

```diff
@@ -26,7 +26,7 @@ def _default_pprint(package_name: str) -> str:
 class OptionsRetriever:
     OPTIONS_TO_DEFAULT = {
         "azure-arm": False,
-        "unbranded": False,
+        "flavor": "azure",
```

Comment:
Default should be non-Azure

Reply:
Default is "non-azure" for tsp in emitter.ts: 
https://github.com/Azure/autorest.python/blob/6e5bf1250d91a143831462b539c3d7c121eb869b/packages/typespec-python/src/emitter.ts#L75-L84

While the logic is both for swagger and tsp, we need to set it "azure" here to avoid influencing generation with swagger. 
I can maker sure `--flavor` works as expected which can also be proved by no-diff in test generation